xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1613,11 +1607,6 @@ int CGUIWindowVideoBase::GetScraperForItem(CFileItem *item, ADDON::ScraperPtr &i
   return foundDirectly ? 1 : 0;
 }
 
-void CGUIWindowVideoBase::OnScan(const std::string& strPath, bool scanAll)
```

Comment:
I agree with montellese here, no need to remove it. better to keep those globals calls in an separate method. Easier to make subsequent changes then.